packages/react-core/src/components/MultipleFileUpload/MultipleFileUploadStatusItem.tsx

```diff
@@ -138,6 +145,14 @@ export const MultipleFileUploadStatusItem: React.FunctionComponent<MultipleFileU
     </span>
   );
 
+  const helperTextVariant = variant === 'danger' ? 'error' : variant;
```

Comment:
@tlabaj  - i wonder if variants should be more uniform across components? Even if in some cases maybe various variants could map to the same result within the component.

Reply:
Good call out @gitdallas.  It may actually make more sense to have the variant  here map to a color. 
@mcoker as proposed such a change to the Banner component. https://github.com/patternfly/patternfly-react/issues/8204.